### ISSUE TITLE
Hide "Hide menu bar icons" when feature is diabled

### ISSUE THREAD

**Issue by pddd** [jacklandrin/OnlySwitch#64]
The icon for "Hide menu bar icons" should disappear, when the feature is disabled/deactivated.
On disabling this feature, hidden icons should be shown.

MacBook Pro 16, M1

<img width="937" alt="image" src="https://user-images.githubusercontent.com/1897661/208377425-914b3391-cf33-4edc-8d06-7d3217cc93d4.png">

**Comment by jacklandrin** [comment]
@pddd You can find Switches Availability on the left of menu bar when settings window is open. Click Disable hide menu bar icons, the icon will disappear. 
![](https://camo.githubusercontent.com/bb87fb662d1d0c6879e7e14df27282d1c21ec2540a08d3a5f33f9b4c10afe676/687474703a2f2f7777772e6a61636b6c616e6472696e2e636f6d2f77702d636f6e74656e742f75706c6f6164732f323032322f30382f53637265656e73686f742d323032322d30382d31362d61742d31302e31312e33352e706e67)

**Comment by adminquest** [comment]
On my System the Menubar don't work - maybe i must enable some rights for only-switch, but which one?

**Comment by jacklandrin** [comment]
@adminquest  Have you checked the option on my last comment's picture?

**Comment by adminquest** [comment]
Thank you very much. I had actually ignored the image. My mistake. Thanks again :-D

**Comment by adminquest** [comment]
<img width="328" alt="image" src="https://user-images.githubusercontent.com/8513401/212529014-aca28f86-c6b1-47a1-b45b-4363cebee4f6.png">
What has been my real problem: If you put the HideSymbol in front of the OnlySwitch (1.) then the button (logically) doesn't work (2.) But you also don't get a hint that you are doing something wrong. A hint why this does not work would be great. If you notice it once, it won't happen again - but a hint would be good.

**Comment by adminquest** [comment]
<img width="922" alt="image" src="https://user-images.githubusercontent.com/8513401/212529144-ab3950af-da8b-4de7-98fc-dfb00beb5326.png">
Maybe here?

**Comment by RealDeemer** [comment]
> but a hint would be good.

I fully agree to that. It took me ages, to find out why that button can't be enabled.
